internal/versions/versions.go

```diff
@@ -249,64 +310,51 @@ func (h *handler) AddReleaseImage(releaseImageUrl, pullSecret, ocpReleaseVersion
 		}
 	}
 
-	// Get minor version
-	ocpVersionKey, err := h.getKey(ocpReleaseVersion)
-	if err != nil {
-		return nil, err
-	}
-
 	// Ensure a relevant OsImage exists
-	osImage, err := h.GetOsImage(ocpVersionKey, cpuArchitecture)
+	osImage, err := h.GetOsImage(ocpReleaseVersion, cpuArchitecture)
 	if err != nil || osImage.URL == nil {
-		return nil, errors.Errorf("No OS images are available for version: %s", ocpVersionKey)
+		return nil, errors.Errorf("No OS images are available for version: %s", ocpReleaseVersion)
 	}
 
 	// Fetch ReleaseImage if exists
-	releaseImage, err := h.GetReleaseImage(ocpVersionKey, cpuArchitecture)
-	if err != nil {
+	releaseImage := funk.Find(h.releaseImages, func(releaseImage *models.ReleaseImage) bool {
+		return *releaseImage.OpenshiftVersion == ocpReleaseVersion && *releaseImage.CPUArchitecture == cpuArchitecture
+	})
+	if releaseImage == nil {
 		// Create a new ReleaseImage
 		releaseImage = &models.ReleaseImage{
-			OpenshiftVersion: &ocpVersionKey,
+			OpenshiftVersion: &ocpReleaseVersion,
 			CPUArchitecture:  &cpuArchitecture,
+			URL:              &releaseImageUrl,
+			Version:          &ocpReleaseVersion,
 		}
-	} else {
```

Comment:
This previously allowed for "updating" was that ever useful?
This is just in-memory, right? Do we ever alter the release images we support outside of startup?

If we do allow this do change, do we still want to be able to update the X.Y key here or will that just always get added as X.Y.Z now? Is it important that that behavior changed?